components/formats-api/src/loci/formats/FormatReader.java

```diff
@@ -1466,6 +1466,15 @@ public void close() throws IOException {
     close(false);
   }
 
+  /**
+   * Get the CoreMetadata corresponding to the current series and resolution
+   *
+   * @return the CoreMetadata
+   */
+  protected CoreMetadata currentCore() {
```

Comment:
Since this helper is technically a getter, should it be called `getCurrentCore` for consistency?

Reply:
Done in d5e6abc.